templates/bundles/EasyAdminBundle/layout.html.twig

```diff
@@ -0,0 +1,8 @@
+{# templates/bundles/EasyAdminBundle/layout.html.twig #}
+
+{% extends '@!EasyAdmin/layout.html.twig' %}
+
+{% block content %}
+    <article class="content overflow-x-scroll">
+    {{ parent() }}
+{% endblock %}
```

Comment:
Aren't we missing an end tag, `</article>`, here?

Reply:
Correct. Thanks